README.md

```diff
@@ -1,32 +1,44 @@
 Node Apps Toolkit
 ===
 
-A collection of helpers and utilities for creating NodeJS Contentful Apps
+The Node Apps Toolkit is a growing collection of helpers, and utilities, for creating NodeJS Contentful Apps.
 
 ## Getting started
  
-You can install this library via
+You can install this library with npm or yarn
  
 ```
 npm install --save @contentful/node-apps-toolkit
+or
+yarn add @contentful/node-apps-toolkit
 ```
 
-and include it in your code like
+## Features
 
+### getManagementToken
+NodeJS Contentful Apps need a management token to interact with Contentful's APIs.
+Creating a management token requires a key pair to be regsitered for the app, follow [this link](http://contentful./developers/docs/references/content-management-api/#/reference/app-keys/app-keys) for more information on key pairs.
```

Comment:
This link isn't currently live.. We could link to a preview directly for the time being if necessary.

Reply:
I would refrain to put the features this into the `README.md`. This codebase generates documentation based on the doc blocks in the code and the reason is that we don't want documentation to live far from the code, otherwise we might forget.

I would move this block [here](https://github.com/contentful/node-apps-toolkit/blob/19ab7acd82e77265f0b44ba5e5b9f9a4b926d3d3/src/keys/get-management-token.ts#L92-L104) and leave the `README.md` with a very silly example which points to the auto-generated and always up to date documentation